ReactiveUI.AndroidSupport/ReactiveDialogFragment.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.ComponentModel;
+using System.Reactive.Subjects;
+using System.Reactive;
+
+namespace ReactiveUI.AndroidSupport
+{
+    /// <summary>
+    /// This is a DialogFragment that is both a DialogFragment and has ReactiveObject powers 
+    /// (i.e. you can call RaiseAndSetIfChanged)
+    /// </summary>
+    public class ReactiveDialogFragment<TViewModel> : ReactiveDialogFragment, IViewFor<TViewModel>, ICanActivate
+        where TViewModel : class
+    {
+        protected ReactiveDialogFragment() { }
```

Comment:
Is there a reason the ctor is protected but the class is not abstract? Same question for the non-generic counterpart.